### ISSUE TITLE
Real world example with mixed bindings

### ISSUE THREAD

**Issue by vinyll** [riot/dom-bindings#5]
Hey Gianluca,

I know this is used for preparing Riot4, and I'd like to use these bricks individually in order to get more involved into the next steps and do a better job interacting and suggesting.

In order to do so I'm trying to get my head into this _dom-binding_ repo to actually tweak and use it.
Reading the tests give me a rough idea of what the concept is but I can't get it work properly.

**Comment by GianlucaGuarini** [comment]
I have documented the [API](https://github.com/riot/dom-bindings#api) could you please be more specific? What's not clear? This is just a template engine not meant to be used to store your components data or handle the life cycle hooks

**Comment by vinyll** [comment]
Indeed I'm not clear.
I'm aware this is not a component library or whatever high end usable lib.

My purpose was to understand what it actually can do; I feel like it's able to condition and loop fragments of an HTML template.

I do have some cases where I'd like to loop nodes and conditionally display some.
Mixing that with [a template library](https://github.com/riot/tmpl/) could give a powerful html template rendering engine.

In short, my need would be to be able to render something like:

```js
template(`
<h1>Todo list</h1>
<ul if={ todos }>
  <li each={ todo in todos } class="{ todo.done && 'done' }">{ todo.label }</li>
</ul>
`, {
  todos: [
    { label: "Saving the world", done: true },
    { label: "Sending men to the moon", done: true },
    { label: "Easy align in CSS", done: false },
    { label: "Understanding JS libs", done: false },
  ]
})
```

**Comment by GianlucaGuarini** [comment]
@vinyll I took the time to make a simple todo list with vanilla riot-dom-bindings. https://codesandbox.io/s/rr3k4pmzrq

Of course you will not write the new riot components in this way because this will be the output of the new compiler. Thanks for asking it

**Comment by vinyll** [comment]
Well, that is a very beautiful and relevant example! Thanks for taking the time to write it, that's a good demo of what this can do 🏅
